recipes/recipes_emscripten/llvm/build.sh

```diff
@@ -9,28 +16,27 @@ export LDFLAGS=""
 
 # Configure step
 emcmake cmake ${CMAKE_ARGS} -S ../llvm -B .         \
-    -DCMAKE_BUILD_TYPE=Release                      \
-    -DCMAKE_PREFIX_PATH=$PREFIX                     \
-    -DCMAKE_INSTALL_PREFIX=$PREFIX                  \
-    -DLLVM_HOST_TRIPLE=wasm32-unknown-emscripten    \
-    -DLLVM_TARGETS_TO_BUILD="WebAssembly"           \
-    -DLLVM_ENABLE_ASSERTIONS=ON                     \
-    -DLLVM_ENABLE_EH=ON                             \
-    -DLLVM_ENABLE_RTTI=ON                           \
-    -DLLVM_INCLUDE_BENCHMARKS=OFF                   \
-    -DLLVM_INCLUDE_EXAMPLES=OFF                     \
-    -DLLVM_INCLUDE_TESTS=OFF                        \
-    -DLLVM_ENABLE_LIBEDIT=OFF                       \
-    -DLLVM_ENABLE_PROJECTS="clang;lld"              \
-    -DLLVM_ENABLE_THREADS=OFF                       \
-    -DLLVM_ENABLE_ZSTD=OFF                          \
-    -DLLVM_ENABLE_LIBXML2=OFF                       \
-    -DLLVM_BUILD_TOOLS=OFF                          \
-    -DCLANG_ENABLE_STATIC_ANALYZER=OFF              \
-    -DCLANG_ENABLE_ARCMT=OFF                        \
-    -DCLANG_ENABLE_BOOTSTRAP=OFF                    \
-    -DCLANG_BUILD_TOOLS=OFF                         \
-    -DCMAKE_CXX_FLAGS="-Dwait4=__syscall_wait4 -fexceptions"
+    -DCMAKE_BUILD_TYPE=Release \
+    -DLLVM_HOST_TRIPLE=wasm32-unknown-emscripten \
+    -DLLVM_ENABLE_ASSERTIONS=ON \
+    -DLLVM_TARGETS_TO_BUILD="WebAssembly" \
+    -DLLVM_ENABLE_LIBEDIT=OFF \
+    -DLLVM_ENABLE_PROJECTS="clang;lld" \
+    -DLLVM_ENABLE_ZSTD=OFF \
+    -DLLVM_ENABLE_LIBXML2=OFF \
+    -DCLANG_ENABLE_STATIC_ANALYZER=OFF \
+    -DCLANG_ENABLE_ARCMT=OFF \
+    -DCLANG_ENABLE_BOOTSTRAP=OFF \
+    -DCMAKE_CXX_FLAGS="-Dwait4=__syscall_wait4 -fexceptions" \
+    -DLLVM_INCLUDE_BENCHMARKS=OFF \
+    -DLLVM_INCLUDE_EXAMPLES=OFF \
+    -DLLVM_INCLUDE_TESTS=OFF \
+    -DLLVM_ENABLE_THREADS=OFF \
+    -DLLVM_BUILD_TOOLS=OFF \
+    -DLLVM_ENABLE_LIBPFM=OFF \
+    -DCLANG_BUILD_TOOLS=OFF \
+    -DLLVM_TABLEGEN=$NATIVE_DIR/llvm-tblgen \
+    -DCLANG_TABLEGEN=$NATIVE_DIR/clang-tblgen \
```

Comment:
I see only the last 2 configs being extra. Can we not change the other configs if possible ?